doc/source/methods/PartialDependenceVariance.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<p>the the -&gt; the (can you search the doc as I think there's a few places this happens)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>